src/core/org/apache/jmeter/report/processor/StatisticsSummaryData.java

```diff
@@ -25,17 +28,15 @@
  *
  */
 public class StatisticsSummaryData {
-
+    private LatencyStats latencyStats = LatencyStats.Builder.create().build();
+    Histogram intervalHistogram = latencyStats.getIntervalHistogram();
+    private Histogram histogram = new Histogram(intervalHistogram);
```

Comment:
What's the precision of the resulting histogram?
What's the memory consumption?

Reply:
This is work in progress. See question asked on dev list also